languages/en.ini

```diff
@@ -1226,6 +1228,8 @@ renew_success = "Renewal Successful"
 renew_success_summary = "Successfully renewed {count, plural, =1 {1 item} other {# items}}."
 Renewed = "Renewed"
 Request full text = "Request full text"
+request_group_fulfillment_method_delivery = "Delivery"
```

Comment:
I'm probably missing some but I don't see where/how these request_group_fulfillment_method* strings are used. Do we still need them for something?

Reply:
Yes, they are used as 'name' params [in the Folio driver](https://github.com/vufind-org/vufind/blob/dev-11.0/module/VuFind/src/VuFind/ILS/Driver/Folio.php#L1623) minus the `request_group_` prefix, which is added in [hold.phtml](https://github.com/vufind-org/vufind/blob/dev-11.0/themes/bootstrap3/templates/record/hold.phtml#L64) to output the request group name.  Hard to search for!

Reply:
Ahh, tricky! Thanks for the clarification. :-)